src/main/resources/schema/ans/0.10.10/utils/variant.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "additionalProperties": false,
+  "description": "Variant metadata describing its ID as well as the websites to which it is assigned",
+  "id": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.10/utils/variant.json",
+  "properties": {
+    "_id": {
+      "$ref": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.10/traits/trait_id.json",
+      "description": "variant ID"
+    },
+    "additional_properties": {
+      "$ref": "https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.10/traits/trait_additional_properties.json"
+    },
+    "content": {
```

Comment:
Can you give an example of what this will look like as JSON data? I'm unclear how this will look in practice.

Reply:
In a general sense, it would be the content type that has variant data.  The immediate case is for story data, so this is an example:

```
{
  "content_elements": [
    {
      "_id": "some_placeholder_ID",
      "content_elements": [
        {
          "content": "Some content zone data",
          "type": "text"
        }
      ],
      "type": "element_group"
    }
  ],
  "promo_items": {
    "basic": {
      "referent": {
        "id": "some image ID",
        "type": "image"
      },
      "type": "reference"
    }
  },
  "taxonomy": {
    "tags": [
      {
        "slug": "some-local-tag",
        "text": "some local tag",
        "type": "tag"
      }
    ]
  },
  "type": "story",
  "version": "0.10.8"
}
```

The logic to assemble the final story would read this story for the overrides and content zones data.

Reply:
Makes sense.  Is there a way to enforce the structure of content with ANS?

Reply:
yes the `required` fields enforces the schema for ANS.

Reply:
Also @cselvaraj  and @hartmanc-wp  I think we need to follow the way `story` had the schema [here](https://github.com/washingtonpost/ans-schema/blob/master/src/main/resources/schema/ans/0.10.9/story.json), so that we exclusively mention what are the fields we want in our variant. 

1. content [here](https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.9/content.json) does not have `content_elements` 
2. And in content there are lot of fields which we are not using [ not a real problem though ]

Reply:
The choice to use the "content" trait was to allow this field to hold different, top-level types.  Instead of it, we could use the JSON schema `oneOf` directive and explicitly list the specific content types that we allow.  That would force us to update when we want to extend variants to other types, but it has the benefit of being more explicit.

Reply:
hey @cselvaraj to the example above you mentioned there are content_elements where in the content object [schema](https://raw.githubusercontent.com/washingtonpost/ans-schema/master/src/main/resources/schema/ans/0.10.9/content.json) there is no such field, so is it because we dont want to enforce content_elements in variant schema?